src/biogeochem/CNCStateUpdate1Mod.F90

```diff
@@ -69,12 +70,11 @@ subroutine CStateUpdateDynPatch(bounds, num_soilc_with_inactive, filter_soilc_wi
        do j = 1,nlevdecomp
           do fc = 1, num_soilc_with_inactive
              c = filter_soilc_with_inactive(fc)
-             cs_soil%decomp_cpools_vr_col(c,j,i_met_lit) = cs_soil%decomp_cpools_vr_col(c,j,i_met_lit) + &
-                  cf_veg%dwt_frootc_to_litr_met_c_col(c,j) * dt
-             cs_soil%decomp_cpools_vr_col(c,j,i_cel_lit) = cs_soil%decomp_cpools_vr_col(c,j,i_cel_lit) + &
-                  cf_veg%dwt_frootc_to_litr_cel_c_col(c,j) * dt
-             cs_soil%decomp_cpools_vr_col(c,j,i_lig_lit) = cs_soil%decomp_cpools_vr_col(c,j,i_lig_lit) + &
-                  cf_veg%dwt_frootc_to_litr_lig_c_col(c,j) * dt
+             do i = i_litr_min, i_litr_max
+                cs_soil%decomp_cpools_vr_col(c,j,i) = &
+                     cs_soil%decomp_cpools_vr_col(c,j,i) + &
+                     cf_veg%dwt_frootc_to_litr_c_col(c,j,i) * dt
+             end do
              cs_soil%decomp_cpools_vr_col(c,j,i_cwd) = cs_soil%decomp_cpools_vr_col(c,j,i_cwd) + &
                   ( cf_veg%dwt_livecrootc_to_cwdc_col(c,j) + cf_veg%dwt_deadcrootc_to_cwdc_col(c,j) ) * dt
```

Comment:
Here, after the litter we have cwd, and I decided to leave it alone for now because it is handled a bit differently.